examples/kafka/kafka-ephemeral-single.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: 1
+    listeners:
+      plain: {}
+      tls: {}
+    config:
+      offsets.topic.replication.factor: 3
+      transaction.state.log.replication.factor: 3
+      transaction.state.log.min.isr: 2
+    storage:
+      type: ephemeral
+  zookeeper:
+    replicas: 1
+    storage:
+      type: ephemeral
```

Comment:
This really doesn't work - you cannot have ephemeral Zoo or Kafka with single node. They will loose all data with every restarts / rolling update. My suggestion is to remove the ephemeral version completely.